datadog_lambda/tracing.py

```diff
@@ -248,91 +247,105 @@ def extract_context_from_sqs_or_sns_event_or_context(
     except Exception:
         logger.debug("Failed extracting context as EventBridge to SQS.")
 
-    try:
```

Comment:
if context is extracted from event bridge, we don't set a checkpoint. Is that expected?